_content/_docs_signals/conditions_script.md

```diff
@@ -64,13 +65,15 @@ To run a stored script, refer to it by using it's id:
 | name | name of this condition. Can be chosen freely. Mandatory. |
 | script_id | The ID of the stored script. Mandatory |
 
-## Accessing the execution context data
```

Comment:
I would recommend to avoid "execution context data" (because it's umständlich) and to avoid "payload" (because it does not make sense). Instead, I'd recommend to consistently use only "runtime data".